views/index.html

```diff
@@ -6,7 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Testing page</title>  
 
-        <link rel="stylesheet" href="../css/fonts.css">
```

Comment:
I put both of these in there with one commented out. Seems to me one will work and one wont.